packages/shared/src/components/cards/CollectionCard/CollectionCard.tsx

```diff
@@ -34,7 +34,7 @@ export const CollectionCard = forwardRef(function CollectionCard(
       return 'line-clamp-3';
     }
 
-    return post.contentHtml ? 'line-clamp-4' : 'line-clamp-9';
+    return post.contentHtml ? 'line-clamp-3' : 'line-clamp-9';
```

Comment:
Only with image should it be 3 from the design.
- title only: 9
- title + description: 4
- title + image: 3

Could you add local test + screenshots for those versions?

Reply:
Definitely. Let me check this.

Reply:
After further checking, the issue is not with the rules inside the code but rather how we check the image. In here, there is no cover image, so it returns the clamp for 4 which is correct, but then we have a process of how we define the image to use which was not utilized here (which will then say we should have used line clamp 3).